### PR TITLE
Going Live

### PR DESCRIPTION
**Pull request by Lockless404**
Made minor changes to allow Github pages to read HTML file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Fully functional on mobile and desktop.
 - No Technologies
 
 ## Live Demo
-https://raw.githack.com/Lockless404/Portfolio/master/portfolio.html
+https://lockless404.github.io/Portfolio/
```
Comment (carlos-ssh) [inline README.md:16]: - Kindly provide a direct link to your GitHub Deploy.
Screen Shot 2021-09-27 at 5 21 26

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Fully functional on mobile and desktop.
...
 
 ## Getting Started
 To get a local copy up and running in your machine:
```
Comment (carlos-ssh) [inline README.md:19]: Your project in desktop version is overlapping in the x-axis kindly solve this issue:
(grey space at your right)
Screen Shot 2021-09-27 at 5 23 53

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Lockless404 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Lockless404) [reply README.md:19]: Thank you for the review. I'm just not sure why the webpage appears like that for you, I've linked two pictures of the webpage on Chrome and Firefox on my PC. It looks different. I'm not sure how to address the problem, any help would e much appreciated. Thank you.

portfolioChrome
portfolioFirefox

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Lockless404, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**